.github/workflows/publish-to-pypi.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   build-and-publish:
-    if: github.repository == 'ni/systemlink-configuration-utility'
+    if: github.repository == 'ni/nisystemlink-configuration'
```

Comment:
Are we also renaming the repo? If so, I believe there are also mentions of it in the CONTRIBUTING.md.

Reply:
I didn't think we were renaming the repo. I only updated the name in places it would be visible from the package. Do you want to rename the repo as well?

Reply:
No, my comment was that if you rename it here, you have to rename the repo as well or the action won't trigger.

Reply:
Oi. Wasn't even paying attention - find-and-replace 🤦‍♂️